docs/cmdline-opts/cert.d

```diff
@@ -14,19 +14,24 @@ Tells curl to use the specified client certificate file when getting a file
 with HTTPS, FTPS or another SSL-based protocol. The certificate must be in
 PKCS#12 format if using Secure Transport, or PEM format if using any other
 engine. If the optional password is not specified, it will be queried for on
-the terminal. Note that this option assumes a \&"certificate" file that is the
-private key and the client certificate concatenated! See --cert and --key to
-specify them independently.
+the terminal if required. Note that this option assumes a \&"certificate"
```

Comment:
It feels a bit superfluous to mention both "optional" and "required" in the same sentence. Maybe remove "optional" here?